### ISSUE TITLE
Why kohya-ss/sd-scripts is not a subrepo

### ISSUE THREAD

**Issue by zetyquickly** [bmaltais/kohya_ss#1678]
Hi,

Seems like this project is hardly backed by [kohya-ss/sd-scripts](https://github.com/kohya-ss/sd-scripts), why isn't it a subrepo of the repo? What are the main reasons cons?

**Comment by rockerBOO** [comment]
I believe this originally was just a simple UI frontend to the scripts. The scripts are not setup like a library so makes it difficult to import it like that. Probably more possible at this current time to import it as a library though since each script has a single function entrypoint that could get arguments passed to it.

**Comment by bmaltais** [comment]
It sort of organically grew. THe original kohya code was in some sort of notes posted on a Japanese site. I created the kohya_ss gui and the author decided to use github once he saw how I was publishing the GUI. We worked together to sort of structure things at his end that was aligned with my structure. Since he used something similar to mine it was easy to just merge his repo into mine when he did updates.

Not the best but it works ;-)
